github-actions/google-internal-tests/lib/main.ts

```diff
@@ -62,8 +62,8 @@ async function main() {
     state: 'pending' as const,
     // The initial waiting status is expected to be overridden by the
     // internal presubmit status service then.
-    description: `Waiting for Google Internal Tests. ${
-      runTestGuideURL ? `@Googlers: See Details for instructions -->` : ''
+    description: `Waiting for tests to run. ${
+      runTestGuideURL ? `@Googlers: Initiate a presubmit. See URL.` : ''
```

Comment:
How about "Waiting for Google tests to start. @Googlers: Initiate a presubmit. Instructions in Details". Or is it too long?

Reply:
Updated to what we discussed offline.